t/t2233-job-info-update.t

```diff
@@ -51,19 +51,39 @@ expiration_add() {
 	echo $new
 }
 
+# Usage: check_expiration FILE VALUE
+# Check and wait for expiration to reach VALUE.
+#
+check_expiration() {
+	local file=$1
+	local value=$2
+	local i=0
+	while (! cat ${file} | jq -e ".execution.expiration == ${value}" \
+		&& [ $i -lt 200 ] )
+	do
+		sleep 0.1
+		i=$((i + 1))
+	done
+	if [ "$i" -eq "200" ]
+	then
+		return 1
+	fi
+	return 0
+}
```

Comment:
I am confused how this works. I'm probably missing something obvious, but `check_expiration` is continuously doing `cat ${file} | jq` until an expected value is observed. However, in the test, nothing I see is updating the file, so how could the contents of the file change? I don't think `update_lookup` is running in the background or anything (maybe just missing it)

Reply:
oh crap ... the answer is I just flubbed this :-)